doc/source/examples/sharp-immersed-boundary/3d-rbf-static-mixer/3d-rbf-static-mixer.rst

```diff
@@ -0,0 +1,297 @@
+#. The remaining boundaries are set as ``noslip`` to emulate the flow in a channel.
+
+.. code-block:: text
+
+  subsection boundary conditions
+    set number = 6
+    subsection bc 0
+      set id   = 0
+      set type = function
+      subsection u
+        set Function expression = 1
+      end
+    end
+    subsection bc 1
+      set id   = 1
+      set type = outlet
+    end
+    subsection bc 2
+      set id   = 2
+      set type = noslip
+    end
+    subsection bc 3
+      set id   = 3
+      set type = noslip
+    end
+    subsection bc 4
+      set id   = 4
+      set type = noslip
+    end
+    subsection bc 5
+      set id   = 5
+      set type = noslip
+    end
+  end
+
+
+Post-Processing
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Pressure drop and flow rate post-processing are enabled to track when steady state is reached and to ensure that mass is preserved. Too high variations between inlet and outlet flow rates are linked to increased error on the pressure drop predictions.
+
+.. code-block:: text
+
+  subsection post-processing
+    set verbosity               = verbose
+    set calculate pressure drop = true
+    set calculate flow rate     = true
+    set inlet boundary id       = 0
+    set outlet boundary id      = 1
+  end
+
+
+-----------------------
+Running the Simulation
+-----------------------
+
+The simulation can be launched on multiple cores using ``mpirun`` and the ``lethe-fluid-sharp`` executable. Using 6 CPU cores, the simulation can be launched with:
+
+.. code-block:: text
+  :class: copy-button
+
+  mpirun -np 6 lethe-fluid-sharp flow_in_long_mixer.prm
+
+
+--------
+Results
+--------
+
+After the simulation has run, streamlines can be used to visualize the pressure and velocity fields through the static mixer, as well as show the mixing effects that can be obtained.
+
++-----------------------------------------------------------------------------------------------------------------------------+
+|  .. figure:: images/long_static_mixer_medium_thick_p_v.png                                                                  |
+|     :align: center                                                                                                          |
+|     :width: 800                                                                                                             |
+|     :name: Streamlines in the static mixer colored by velocity magnitude and pressure                                       |
+|                                                                                                                             |
+|     Streamlines in the static mixer colored by velocity magnitude and pressure                                              |
+|                                                                                                                             |
++-----------------------------------------------------------------------------------------------------------------------------+
+
```

Comment:
Maybe a figure for the pressure drop and flow rat could be interesting (to see the mass conservation maybe)

Reply:
I added a figure and a few lines for this